themes/default/subtemplates/admin.inc.tpl

```diff
@@ -1,1152 +1,1166 @@
-<td class="d"><p><input id="upload_images_all" type="radio" name="upload_images" value="3"{if $edSet.upload_images==3} checked="checked"{/if} /><label for="upload_images_all">{#upload_enabled_all#}</label><br />
-<input id="upload_images_users" type="radio" name="upload_images" value="2"{if $edSet.upload_images==2} checked="checked"{/if} /><label for="upload_images_users">{#upload_enabled_users#}</label><br />
-<input id="upload_images_admins_mods" type="radio" name="upload_images" value="1"{if $edSet.upload_images==1} checked="checked"{/if} /><label for="upload_images_admins_mods">{#upload_enabled_admins_mods#}</label><br />
-<input id="upload_images_disabled" type="radio" name="upload_images" value="0"{if $edSet.upload_images==0} checked="checked"{/if} /><label for="upload_images_disabled">{#disabled#}</label></p>
-{assign var="settings_upload_max_width" value=$edSet.upload_max_img_width}
-{assign var="input_upload_max_width" value="<input id=\"upload_max_img_width\" type=\"text\" name=\"upload_max_img_width\" value=\"$settings_upload_max_width\" size=\"5\" />"}
-{assign var="settings_upload_max_height" value=$edSet.upload_max_img_height}
-{assign var="input_upload_max_height" value="<input type=\"text\" name=\"upload_max_img_height\" value=\"$settings_upload_max_height\" size=\"5\" />"}
-{assign var="settings_upload_max_img_size" value=$edSet.upload_max_img_size}
-{assign var="input_upload_max_filesize" value="<input type=\"text\" name=\"upload_max_img_size\" value=\"$settings_upload_max_img_size\" size=\"5\" />"}
-<p><label for="upload_max_img_width">{#max_upload_size#|replace:"[width]":$input_upload_max_width|replace:"[height]":$input_upload_max_height|replace:"[filesize]":$input_upload_max_filesize}</label></p></td>
-</tr>
-<tr>
-<td class="c"><strong>{#autolink#}</strong><br /><span class="small">{#autolink_desc#}</span></td>
-<td class="d"><input id="autolink" type="checkbox" name="autolink" value="1"{if $edSet.autolink==1} checked="checked"{/if} /><label for="autolink">{#autolink_enabled#}</label></td>
-</tr>
-<tr>
-<td class="c"><strong>{#count_views#}</strong><br /><span class="small">{#count_views_desc#}</span></td>
-<td class="d"><input id="count_views" type="checkbox" name="count_views" value="1"{if $edSet.count_views==1} checked="checked"{/if} /><label for="count_views">{#views_counter_enabled#}</label></td>
-</tr>
-<tr>
-<td class="c"><strong>{#rss_feed#}</strong><br /><span class="small">{#rss_feed_desc#}</span></td>
-<td class="d"><input id="rss_feed" type="checkbox" name="rss_feed" value="1"{if $edSet.rss_feed==1} checked="checked"{/if} /><label for="rss_feed">{#rss_feed_enabled#}</label></td>
-</tr>
-
-<tr>
-<td class="c"><strong>{#threads_per_page#}</strong><br /><span class="small">{#threads_per_page_desc#}</span></td>
-<td class="d"><input type="text" name="threads_per_page" value="{$edSet.threads_per_page|escape}" size="5" /></td>
-</tr>
+{config_load file=$language_file section="admin"}

```

Comment:
Why does the complete file seems to be altered? What did go wrong here?

Reply:
something like your editor converting DOS file endings to UNIX ones (or the reverse). No way to tell without the actual, byte-for-byte accurate files before/after